### ISSUE TITLE
First Application Walkthrough broken example

### ISSUE THREAD

**Issue by the7erm** [slimphp/Slim-Website#269]
TLDR; It looks like code blocks are being wrapped in something like `{strip}{/strip}` in smarty.

FYI, I tried viewing the page in firefox & chrome.

When reading the first app [walkthrough](https://www.slimframework.com/docs/tutorial/first-app.html)

In the `Create The Application` section the example code is missing new lines.
![image](https://user-images.githubusercontent.com/2530157/34179343-bc080366-e4c7-11e7-8d30-fafac34c542c.png)
This is what I'm getting in the console:
![image](https://user-images.githubusercontent.com/2530157/34179554-a582fc76-e4c8-11e7-913a-9f4449cc9ea9.png)

I clicked edit this page and then viewed the raw version.  Everything looks fine.
https://raw.githubusercontent.com/slimphp/Slim-Website/gh-pages/docs/tutorial/first-app.md
```
### Create The Application

There's a really excellent and minimal example of an `index.php` for Slim Framework on the [project homepage](http://www.slimframework.com) so we'll use that as our starting point.  Put the following code into `src/public/index.php`:

{% highlight php %}
<?php
use \Psr\Http\Message\ServerRequestInterface as Request;
use \Psr\Http\Message\ResponseInterface as Response;

require '../vendor/autoload.php';

$app = new \Slim\App;
$app->get('/hello/{name}', function (Request $request, Response $response) {
    $name = $request->getAttribute('name');
    $response->getBody()->write("Hello, $name");

    return $response;
});
$app->run();

{% endhighlight %}
```

So I viewed the source.
```
<figure class="highlight"><pre><code class="language-php" data-lang="php"><span class="cp">&lt;?php</span><br data-jekyll-commonmark-ghpages="" /><span class="k">use</span> <span class="nx">\Psr\Http\Message\ServerRequestInterface</span> <span class="k">as</span> <span class="nx">Request</span><span class="p">;</span><br data-jekyll-commonmark-ghpages="" /><span class="k">use</span> <span class="nx">\Psr\Http\Message\ResponseInterface</span> <span class="k">as</span> <span class="nx">Response</span><span class="p">;</span><br data-jekyll-commonmark-ghpages="" /><br data-jekyll-commonmark-ghpages="" /><span class="k">require</span> <span class="s1">'../vendor/autoload.php'</span><span class="p">;</span><br data-jekyll-commonmark-ghpages="" /><br data-jekyll-commonmark-ghpages="" /><span class="nv">$app</span> <span class="o">=</span> <span class="k">new</span> <span class="nx">\Slim\App</span><span class="p">;</span><br data-jekyll-commonmark-ghpages="" /><span class="nv">$app</span><span class="o">-&gt;</span><span class="na">get</span><span class="p">(</span><span class="s1">'/hello/{name}'</span><span class="p">,</span> <span class="k">function</span> <span class="p">(</span><span class="nx">Request</span> <span class="nv">$request</span><span class="p">,</span> <span class="nx">Response</span> <span class="nv">$response</span><span class="p">)</span> <span class="p">{</span><br data-jekyll-commonmark-ghpages="" />    <span class="nv">$name</span> <span class="o">=</span> <span class="nv">$request</span><span class="o">-&gt;</span><span class="na">getAttribute</span><span class="p">(</span><span class="s1">'name'</span><span class="p">);</span><br data-jekyll-commonmark-ghpages="" />    <span class="nv">$response</span><span class="o">-&gt;</span><span class="na">getBody</span><span class="p">()</span><span class="o">-&gt;</span><span class="na">write</span><span class="p">(</span><span class="s2">"Hello, </span><span class="nv">$name</span><span class="s2">"</span><span class="p">);</span><br data-jekyll-commonmark-ghpages="" /><br data-jekyll-commonmark-ghpages="" />    <span class="k">return</span> <span class="nv">$response</span><span class="p">;</span><br data-jekyll-commonmark-ghpages="" /><span class="p">});</span><br data-jekyll-commonmark-ghpages="" /><span class="nv">$app</span><span class="o">-&gt;</span><span class="na">run</span><span class="p">();</span></code></pre></figure>
```

It looks like whatever is importing that text is removing new lines.  If you're using the smarty template engine I'd guess the text is being wrapped in `{strip}{/strip}` tags.

Please fix this so new users can learn how to use slim.

Thank you.

**Comment by jonsa** [comment]
I was just about to report this.

From what I can tell some post-processing of the source is removing the generated ```<br data-jekyll-commonmark-ghpages="" />``` tags.

It happens when
```html
<!-- formatted for readability -->
<figure class="highlight">
    <pre>
        <code class="language-php" data-lang="php">
            <span class="cp">&lt;?php</span><br data-jekyll-commonmark-ghpages="" />
            <span class="k">use</span>
            <span class="nx">\Psr\Http\Message\ServerRequestInterface</span>
            <span class="k">as</span>
            <span class="nx">Request</span>
            <span class="p">;</span>
            <br data-jekyll-commonmark-ghpages="" />
            ...
        </code>
    </pre>
</figure>
```
is turn into this
```html
<!-- formatted for readability -->
<figure class="highlight">
    <pre class=" language-php">
        <code class=" language-php" data-lang="php">
            <span class="token delimiter">&lt;?php</span>
            <span class="token keyword">use</span>
            <span class="token package">
                <span class="token punctuation">\</span>
                Psr
                <span class="token punctuation">\</span>
                Http
                <span class="token punctuation">\</span>
                Message
                <span class="token punctuation">\</span>
                ServerRequestInterface</span>
                <span class="token class-name">as</span>
                Request
                <span class="token punctuation">;</span>
            </span>
            ...
        </code>
    </pre>
</figure>
```

**Comment by akrabat** [comment]
If you can work out what's going wrong, we'd love a fix. It's just standard Jekyll as far as I know.

**Comment by mathmarques** [comment]
It's not only on First Application, it's the whole doc.
GitHub pages uses always the last version of jekyll and it's plugins

Something updated here: https://github.com/slimphp/Slim-Website/pull/249/commits/49d2745de3c8573141dbc1fa2cd74bed075fb9da (the last version) broke the highlight.

If you build the page locally with https://github.com/slimphp/Slim-Website/pull/249/commits/c8355836d5b90669968830956ef53dadea60777a everything should work as expected.

Something is changing `\n`(new line) to `<br data-jekyll-commonmark-ghpages="" />` 
See the generated HTML [before the update](https://gist.github.com/mathmarques/1ebfa57108e5ae56c750ee44a18679b8) and [after the update](https://gist.github.com/mathmarques/7d7fd72131a0fec271345f8fc47bc878)

I haven't found the solution yet.

**Comment by mathmarques** [comment]
Changing the highlight code from :
`{% highlight php %}`
to
\`\`\`php
Solve the problem(using rouge through markdown).
But \`\`\`php don't work inside  the `<figure>` tag... I don't know how to fix this yet

**Comment by the7erm** [comment]
Thanks for all your hard work.
